.gitignore

```diff
@@ -6,4 +6,4 @@ old.txt
 *.svg
 *.test
 vendor/
-*.xml
+*.xml
```

Comment:
There is still a change to the trailing newline here. Instead of changing the file again, let's just drop the original two commits on your branch and checkout the original version of this file.